cranelift/codegen/src/isa/aarch64/inst.isle

```diff
@@ -833,6 +833,15 @@
         (not_taken BranchTarget)
         (kind CondBrKind))
 
+       ;; A conditional branch which tests the `bit` of `rn` and branches
+       ;; depending on `kind`.
+       (TestBranch
```

Comment:
Maybe name this `TestBitAndBranch`? Because `cbz` is already "test (compare) and branch" and the unique thing about `tbz` is that it is testing a bit.